controller/controller-board/src/app/controllers/TaskController.js

```diff
@@ -0,0 +1,95 @@
+(function(){
+
+    angular
+        .module('app')
+        .controller('TaskController', [
+            'taskService',
+            TaskController
+        ]);
+
+    function TaskController(taskService) {
+        var vm = this;
+        vm.allTasks = [];
+        vm.showDetail = [];
+        vm.loadComplete = false;
+        vm.statusCode = -1;
+        vm.errorMessage = '';
+
+        taskService.getAllTasks()
+            .then(function(result) {
+                vm.statusCode = result.status;
+                vm.allTasks = result.data;
+                // todo: remove fake tasks below
+                vm.allTasks.push({
+                    "name": "com.pinterest.rocksplicator.controller.tasks.HealthCheckTask",
+                    "priority": 1,
+                    "body": "{\"numReplicas\":3,\"intervalSeconds\":0}",
+                    "id": 13,
+                    "state": 0,
+                    "clusterName": "my_cluster",
+                    "createdAt": 1500529716000,
```

Comment:
what are these values?

Reply:
These are fake tasks values so that there could be some tasks that are currently in running state for front end display. I will remove them after we don't need them anymore .